### ISSUE TITLE
CCLE Callset Individuals missing

### ISSUE THREAD

**Issue by kellrott** [bmeg/bmeg-etl#91]
From ccle.Biosample.Edge.json:

```
{
	"gid": "(biosample:ccle:2313287_STOMACH)--sampleOf->(individual:)",
	"fromLabel": "Biosample",
	"from": "biosample:ccle:2313287_STOMACH",
	"label": "sampleOf",
	"toLabel": "Individual",
	"to": "individual:",
	"data": {}
}
```

**Comment by kellrott** [comment]
Still broken in most recent build

**Comment by prismofeverything** [comment]
To be clear, there are no CCLE Individual vertexes yeah? So these just need to not be generated?

**Comment by adamstruck** [comment]
Fixed in #192
